db/migrate/20200417174333_add_unique_index_to_hosts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexToHosts < ActiveRecord::Migration[5.2]
+  def change
+    add_index(:hosts, %i[name account_id], unique: true)
```

Comment:
I don't think that's right - you can have multiple hosts with the same name in the same account, ID is the only field that should be unique for hosts. We actually have a few of these already in QA at least